src/users/users.controller.ts

```diff
@@ -78,6 +105,8 @@ export class UsersController {
   }
 
   @Delete(`${UsersRoutes.delete}/:userId`)
+  @HttpCode(HttpStatus.NO_CONTENT)
```

Comment:
ici je suppose que tu voulais mettre "HttpStatus.OK" au lieux de "HttpStatus.NO_CONTENT"

Reply:
Les deux vont bien pour les requêtes. Sauf que OK - signifie que tout s'est bien passés, et NO_CONTENT signifie que ca s'est bien passés, et que je n'attends pas de contenu. [Check cette ref](https://www.google.com/search?q=what+is+the+http+status+for+data+deletion+%3F&oq=w&aqs=chrome.1.69i60j69i59l3j69i57j69i60l3.10441j0j1&sourceid=chrome&ie=UTF-8).

Reply:
anh super je vois, je ne savais pas pour le NO_CONTENT

Reply:
> anh super je vois, je ne savais pas pour le NO_CONTENT

Voilà une référence StackOverflow où on explque le principe : [Link](https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete#:~:text=For%20a%20DELETE%20request%3A%20HTTP,but%20not%20fully%20applied%20yet.)

Reply:
j'ai lu la ref, je comprends mieux merci